sync-todo/v2/client/flutter/lib/screens/log_in.dart

```diff
@@ -41,7 +43,7 @@ class _LogInState extends State<LogIn> {
             child: Column(
               children: [
                 Text(_isLogin ? 'Log In' : 'Sign Up', style: const TextStyle(fontSize: 25)),
-                loginField(_emailController, labelText: "Email", hintText: "Enter valid email like abc@gmail.com"),
+                loginField(_emailController, labelText: "Email", hintText: "Enter valid email (e.g. abc@gmail.com)"),
```

Comment:
Style guide: we generally avoid Latin words and phrases. Could we replace with "For example"?